inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -595,7 +595,7 @@ obs_end   <- ISOweek2date(sprintf("%s-7", reporting_week))
 
 ### Person
 
-* [Who is affected: how many in total; male or female; young, adult or old? What are the links between affected people – work place, school, social gathering?  Is there a high rate of illness in contacts?  Is there a high rate of illness in health workers? You may want to include:  a bar chart showing case numbers or incidence by age group and sex; attack rates (AR); and numbers of deaths (in suspected and confirmed cases), mortality rates and/or case fatality ratio (CFR)]  
+* [Who is affected: how many in total; male or female; young, adult or old? What are the links between affected people – work place, school, social gathering?  Is there a high rate of illness in contacts?  Is there a high rate of illness in a specific area? You may want to include:  a bar chart showing case numbers or incidence by age group and sex; attack rates (AR); and numbers of deaths (in suspected and confirmed cases), mortality rates and/or case fatality ratio (CFR)]  
```

Comment:
health workers more appropriate here (in a specific area would fit in the place sub-heading)